com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -729,7 +729,7 @@ internal static void HandleNetworkVariableDeltas(List<INetworkVariable> networkV
                     long readStartPos = stream.Position;
 
                     networkVariableList[i].ReadDelta(stream, IsServer, localTick, remoteTick);
-                    PerformanceDataManager.Increment(ProfilerConstants.NetworkVarReceived);
+                    PerformanceDataManager.Increment(ProfilerConstants.NetworkVarDeltas);
```

Comment:
coming from: #632 — fix: Correcting the profiler names for network vars to be appropriately named.